### PR TITLE
NEPT-107: Revert "feat(communities) Remove duplicated step caused by the merge …

### PR DESCRIPTION
**Pull request by Fefaine**
…from communities to 2.3"

This reverts commit 9dc010af4e451128858ff59b527d9a99eba3ce7d.

### CODE REVIEW

**Comment by netlooker** [comment]
👍

File: tests/features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -569,60 +569,7 @@ protected function getNodeByTitle($type, $title) {
...
-    foreach ($rows as $row) {
-      if (strpos($row->getText(), $search) !== FALSE) {
-        return $row;
-      }
-    }
-    throw new \Exception(sprintf('Failed to find a row containing "%s" on the page %s', $search, $this->getSession()
-      ->getCurrentUrl()));
-  }
-
+
```
Comment (voidtek) [inline tests/features/bootstrap/FeatureContext.php]: This line does not pass the phpcs.

**Comment by voidtek** [comment]
:+1:

**Comment by drupol** [comment]
👍

**Comment by Fefaine** [comment]
Youhou

**Pull request merged**